check.go

```diff
@@ -805,7 +812,7 @@ func (s *store) Repair(report CheckReport, options *RepairOptions) []error {
 }
 
 // compareFileInfo returns a string summarizing what's different between the two checkFileInfos
-func compareFileInfo(a, b checkFileInfo, ignoreChownErrors bool) string {
+func compareFileInfo(a, b checkFileInfo, ignoreChownErrors, ignoreTimestamps, ignorePermissions bool) string {
 	if a.typeflag == b.typeflag && a.uid != b.uid && a.gid != b.gid && a.size != b.size &&
 		(os.ModeType|os.ModePerm)&a.mode != (os.ModeType|os.ModePerm)&b.mode {
```

Comment:
Shouldn’t most of the `!=`s here be `==`?

Reply:
Yes, I think so.